CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 1.0.1 (Next)
 
+* [#23](https://github.com/dblock/strava-ruby-client/pull/23): Fixed handling of create_upload errors - [@ylecuyer](https://github.com/ylecuyer).
```

Comment:
Since you finished this, add yourself, `, [simonneutert](https://github.com/simonneutert)`.